CHANGES.txt

```diff
@@ -1,3 +1,77 @@
+2019-09-03 version 3.10.0 (C++/Java/Python/PHP/Objective-C/C#/Ruby/JavaScript)
+
+  C++
+  * Switch the proto parser to the faster MOMI parser.
+  * Properly escape Struct keys in the proto3 JSON serializer.
+  * Inlined implementation of ..._IsValid routines for enums with small numbers of distinct ranges of values
```

Comment:
This enum _IsValid optimization ended up getting rolled back and so it is not included in this release.

Reply:
Done